installation-bundle/src/InstallTool.php

```diff
@@ -191,12 +184,9 @@ public function hasOldDatabase(): bool
      */
     public function hasConfigurationError(array &$context): bool
     {
-        $row = $this->connection
-            ->query('SELECT @@version as Version')
-            ->fetch(\PDO::FETCH_OBJ)
-        ;
+        $row = $this->connection->fetchAssociative('SELECT @@version as Version');
 
-        [$version] = explode('-', $row->Version);
+        [$version] = explode('-', $row['Version']);
```

Comment:
```suggestion
        [$version] = explode('-', $this->connection->fetchOne('SELECT @@version'));
```

Reply:
Changed in efd72b05bed5fb7afde25e4688f9d878c79bf440.